doc/content/guides/getting-started/console/create-gateway/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Create Gateway"
+description: ""
+weight: 2
+---
+
+Learn how to register a gateway using the Console.
+
+<!--more-->
+
+## Create Gateway
+
+Go to **Gateways** in the top menu, and click **+ Add Gateway** to reach the gateway registration page. Fill the gateway ID, gateway EUI (if your gateway has an EUI) and frequency plan. The other fields are optional. Click **Create Gateway** to create the gateway.
+
+{{< figure src="gateway-creation.png" alt="Gateway creation" >}}
+
+Your gateway will be created and you will be redirected to the gateway overview page of your newly created gateway.
+
+{{< figure src="gateway-overview.png" alt="Gateway overview" >}}
+
+You can now connect your gateway to {{% tts %}}.
+
+### Create Gateway API Key
```

Comment:
Hijacking to mention: There are a few gateway connection guides that link to this part of the initial doc (e.g. https://github.com/TheThingsNetwork/lorawan-stack/blob/master/doc/content/guides/connecting-gateways/nasysoutdoorgateway/_index.md#registration)

The links in those files should be updated as well. There may be more references else-where

Reply:
Good catch, thanks!